userspace/falco/falco_outputs.cpp

```diff
@@ -146,6 +146,12 @@ void falco_outputs::handle_event(gen_event *ev, string &rule, string &source,
 
 	std::lock_guard<std::mutex> guard(m_ls_semaphore);
 	lua_getglobal(m_ls, m_lua_output_event.c_str());
+	char hostname[1024];
+	int err = gethostname(hostname, sizeof(hostname));
```

Comment:
While this might be good for most use cases, imagine this scenario.

You run this as a Daemonset on Kubernetes.

Then you consume the outputs somewhere else and try to get the hostname. 

You notice that you are getting the hostname of the container and not the one of the node.

My proposed solution, is to create a new environment variable for the hostname. Then we read that environment variable and if it's not present we get the system hostname.

In that way, when deploying in kubernetes we can just go and do something like this in the Pod spec:

```yaml
...
      containers:
        - name: falco
          image: falcosecurity/falco:latest
          env:
            - name: FALCO_GRPC_HOSTNAME
              valueFrom:
                fieldRef:
                  fieldPath: spec.nodeName
...
```

Please note that `FALCO_GRPC_HOSTNAME` is just a filler idea I had for the variable name, I'm totally unsure what name to pick !

Reply:
I've updated the PR to reflect that change

Reply:
Let's merge the functionality then we can pick battle on names (I really like such battles Lorenzo knows ahaha).